src/main.js

```diff
@@ -17,3 +23,25 @@ document.addListenerToButtons = addListenerToButtons(
     'what starship from Star Wars is showed in the picture.',
   ]
 );
+
+const intervalId = setInterval(() => {
+  findLastElementPosition();
+}, 1000);
+
+document.addEventListener('keydown', (event) => {
+  if (document.querySelector('.intro__container') && event.key === 'Escape') {
+    hideStarWarsIntro();
+    clearInterval(intervalId);
+  }
+});
```

Comment:
You could also add a way to remove both event listeners in this file once they are no longer needed to prevent memory leaks by passing `{once: true}` as the third argument of `addEventListener`.

Reply:
When I tried to add once: true to the 'touchstart' listener then it doesn't work. For 'keydown' works great.

Reply:
Right. My bad. You set the closing of the intro to a double tap so if an event fires only once there is no way of checking if there was a second tap. Please consider removing the event listener after a second tap by using `removeEvent.